setup.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+
+RED='\033[0;31m'
+NC='\033[0m'
+python_command="python"
+pip_command="pip"
+
+function checkPythonVersion () {
+    case "$($1 --version 2>&1)" in
+    *" 3."*)
+        return 1
+        ;;
+    *)
+        return 0
+        ;;
+    esac
+}
+
+function checkPIPVersion () {
+    case "$($1 --version 6>&1)" in
+    *" 3."*)
+        return 1
+        ;;
+    *)
+        return 0
+        ;;
+    esac
+}
+
+function main () {
+    if checkPythonVersion $python_command; then
+        python_command="python3"
+        if checkPythonVersion $python_command; then
+            echo "Python3 not found."
+            exit 1
+        fi
+        echo "Using python3 instead of python"
+    fi
+
+    if checkPIPVersion $pip_command; then
```

Comment:
I just noticed `checkPythonVersion` and `checkPIPVersion` is basically the same function. We can probably merge them.